### DISCUSSION TITLE
New Projects in Github Octernships

### DISCUSSION THREAD

**Discussion by firoz3130** [community/community#53607] [GitHub Education]
When Will new projects start as part of Github octernships?
Do we have any specific timeline for this?

**Comment by FireCoder1** [comment]
Up , please  if anyone knows the answer , please inform us

**Comment by ninadsutrave** [comment]
Yes please

**Comment by Subha-5** [comment]
Yes please, we are still waiting for new projects

**Comment by thunder-007** [comment]
Yes

**Comment by akshayamadhuri** [comment]
@firoz3130 are the projects out?

**Comment by jaiakash** [comment]
Check now, projects are out. https://education.github.com/globalcampus/octernships

**Comment by akshayamadhuri** [comment]
can anyone explain what's the process of octernship do we need to work on the project after conformation of our application or should we start just contributing

**Reply by jaiakash** [reply]
Each org have a task in octernship page, you have to apply on octernship page and accordingly complete the task. If the mentor seems impressed, then they will take you in for the octernship

**Comment by firoz3130** [comment]
Did anyone get any reply from the mentors?I have completed the assignment before the deadline and had pushed the code in the repo they have created for me. The deadline was on June 19. And the octernship starts at July 3rd. But still I didnt receive any response from the mentors.
Anybody knows What I have to do? @FireCoder1 @ninadsutrave @jaiakash @akshayamadhuri @Thunder-007

**Comment by mestir** [comment]
[]()

**Comment by akshayamadhuri** [comment]
@firoz3130 I am waiting too

**Reply by firoz3130** [reply]
I still didn't receive any updates/feedback @akshayamadhuri
